figures/scripts/fig3_figS4-mutational-signatures-panels.R

```diff
@@ -135,10 +151,10 @@ samples_in_order <- signatures_results %>%
   arrange(-exposure) %>%
   pull(Kids_First_Biospecimen_ID)
 
-sample_exposures_barplot <- signatures_results %>%
+signatures_results_fct <- signatures_results %>%
   mutate(Kids_First_Biospecimen_ID = factor(Kids_First_Biospecimen_ID,
-                                            levels = samples_in_order)) %>%
-  ggplot() +
+                                            levels = samples_in_order))
```

Comment:
Removing piping into `ggplot` gives me just a bit of joy.

Reply:
I knew you'd like that! There will probably be more of it!